### ISSUE TITLE
Enabling basic auth for local & prod environments doesn't work

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-security#84]
Steps to reproduce:
1. Follow the docs to add local environment specific config for php basic auth
2. Load the site
3. Basic auth not requested / required

There should be a prompt to enter the username and password.

I think the way environment checks are incorporated into the plugin is very confusing. Environments should be opted in to explicitly by type or globally as required.

In the meantime though the plugin integration should be fixed so that the docs are accurate.

Acceptance criteria:

- [ ] ...

**Comment by rmccue** [comment]
@roborourke what's the deal with this? Still an issue?

**Comment by roborourke** [comment]
Yeah it is, we should fix it but I think it might be a breaking change, needs a little research to clear up the AC
